Baby Monitor/Source Files/Modules/Settings/ServerSettingsViewModel.swift

```diff
@@ -8,15 +8,20 @@ import RxSwift
 import RxCocoa
 
 final class ServerSettingsViewModel: BaseSettingsViewModelProtocol {
-    
+
+    let analytics: AnalyticsManager
```

Comment:
I see that analytics are defined in many ViewModels, you could create a protocol like AnalyticsViewModel with this property.

Reply:
That's true but I want to inject analytics from appDependencies, so it would be one place where it is created. Do you have some idea how can be this achieved?